composer.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "algolia/algoliasearch-client-php-helper",
+  "description": "Algolia extension for search API Client",
+  "keywords": ["algolia", "search", "api", "client", "php", "helper"],
+  "type": "library",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Algolia Team",
+      "email": "contact@algolia.com"
+    }
+  ],
+  "require": {
+    "php": "^5.6 || ^7.0",
+    "algolia/algoliasearch-client-php": "^2.2"
+  },
+  "suggest": {
+    "ext-dom": "Required to use the HTML Splitter."
+  },
+  "require-dev": {
+    "friendsofphp/php-cs-fixer": "^2.15.1",
+    "fzaninotto/faker": "^1.8",
+    "phpunit/phpunit": "^5.7.27"
+  },
+  "autoload": {
+    "psr-4": {
+      "Algolia\\AlgoliaSearch\\Helper\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "Algolia\\AlgoliaSearch\\Helper\\Tests\\": "tests/"
+    }
+  },
+  "scripts": {
```

Comment:
Can we make this similar as we have on the @chloelbn pull request on quality tools?